builtins/pcre/CMakeLists.txt

```diff
@@ -13,13 +13,19 @@ foreach(var PCRE_FOUND PCRE_VERSION PCRE_INCLUDE_DIR PCRE_PCRE_LIBRARY PCRE_LIBR
 endforeach()
 
 if(WIN32)
-  if(winrtdebug)
-    set(PCRE_POSTFIX $<$<CONFIG:Debug>:d>)
-    set(pcre_config_kind "Debug")
+  if(CMAKE_GENERATOR MATCHES Ninja)
+    if (CMAKE_BUILD_TYPE MATCHES Debug)
```

Comment:
```suggestion
  if(CMAKE_GENERATOR MATCHES Ninja AND CMAKE_BUILD_TYPE MATCHES Debug)
```
?

Reply:
Nope. the `if()` - `else()` test is for ninja only, independently of the build type